jetson/science_bridge/src/__main__.py

```diff
@@ -201,103 +214,79 @@ def heater_transmit(self, channel, msg):
         # Upon Receiving a heater on/off command, send a command for the appropriate mosfet
         print("Heater cmd callback")
         struct = Heater.decode(msg)
-        message = "$Mosfet,{device},{enable},"
-
-        # Heater/Nichromes mosfet devices 7, 8, and 9, so add 7 to get the appropriate mosfet
 
         translated_device = Heater_Map[struct.device]
-        message = message.format(device=translated_device,
-                                 enable=int(struct.enabled))
-
-        message = self.add_padding(message)
-        print(message)
-        self.ser.close()
-        self.ser.open()
-        if self.ser.isOpen():
-            self.ser.write(bytes(message, encoding='utf-8'))
-        pass
+        message = self.format_mosfet_message(translated_device, int(struct.enabled))
+        self.uart_send(message)
 
     def heater_auto_transmit(self, channel, msg):
         # Send the nucleo a message telling if auto shutoff for heaters is off or on
         struct = HeaterAutoShutdown.decode(msg)
         message = "$AutoShutoff,{enable}"
```

Comment:
can these be all caps? like $AUTONSHUTOFF and $MOSFET for consistency??

Reply:
my understanding of reading the code was that all caps was for msgs from nucleo to jetson. normal casing was for jetson to nucleo. but yea i can change it eventually (it'll break things if I don't reflash the nucleo though).

Reply:
no...it was always supposed to be all caps for everything, it was just smth never fixed last year since the code never underwent a formal review lol i think it looks cleaner in all the same format, if you'd prefer to keep it this was that's fine, can you add a comment in the readme about it though?

Reply:
i'll change it to all caps eventually